templates/react-ecs/package.json

```diff
@@ -2,14 +2,14 @@
   "name": "mud-template-react-ecs",
   "private": true,
   "scripts": {
-    "build": "pnpm recursive run build",
-    "dev": "mprocs",
+    "build": "forge || export PATH=$PATH:~/.foundry/bin; pnpm recursive run build",
+    "dev": "forge || export PATH=$PATH:~/.foundry/bin; mprocs",
```

Comment:
i think this would feel cleaner if we moved `forge || export PATH=$PATH:~/.foundry/bin;` into a separate pnpm script that we then use in the places where we need forge

Reply:
It doesn't work like that. We need to change `$PATH`, but if we run a separate pnpm script, it runs in a different shell, so the changes to `$PATH` don't propagate to where we need them. We could create a separate script file and just do `". <new script>.sh; mprocs"` if you think that's cleaner, the `.` (or `source`, depending on the shell used) runs the script in the existing shell - but you need a shell script for that, not a pnpm script.